backend/src/main/java/com/htecgroup/skynest/model/dto/FolderDto.java

```diff
@@ -0,0 +1,32 @@
+package com.htecgroup.skynest.model.dto;
+
+import com.htecgroup.skynest.model.entity.BucketEntity;
+import com.htecgroup.skynest.model.entity.FolderEntity;
+import com.htecgroup.skynest.model.entity.UserEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.With;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class FolderDto {
```

Comment:
must we have more dtos

Reply:
Well that's how we worked so far, I'm just keeping up consistency.